### ISSUE TITLE
Dark theme for widgets

### ISSUE THREAD

**Issue by dan0xii** [stefan-niedermann/nextcloud-notes#460]
What's everybody's thoughts on this dark theme style for the widgets? I'm having to re-work the app dark theme for another issue and this has come out of it.
Single note widget:
![screenshot_1535473148](https://user-images.githubusercontent.com/1926270/44745274-280b1f00-aaff-11e8-8103-60ae277ccdd3.png)

Note list widget:
![screenshot_1535471384](https://user-images.githubusercontent.com/1926270/44745290-35280e00-aaff-11e8-9c15-2202cc4feb2b.png)

![screenshot_1535473326](https://user-images.githubusercontent.com/1926270/44745298-39ecc200-aaff-11e8-852e-6bcb67985eed.png)

**Comment by stefan-niedermann** [comment]
Do we really need an option here? Why not simply use the global theme setting for every widget? How many people will set their theme to `dark` and have white widgets on their home screen? Otherwise: Great, the widgets look good when themed :)

**Comment by dan0xii** [comment]
I thought about that and came to the conclusion that "there's always one". :)

I agree though and it would simplify matters quite a bit. I'll remove the option for the first version.

**Comment by jancborchardt** [comment]
Yep, first we should always go for the simple version. :) Good call @stefan-niedermann @dan0xii! :)

By the way – not trying to insert complication here – what do you think about slight transparency for the widgets? I find that this makes them instantly look much much nicer on the screen because they are more integrated with your background. It could be just a slight transparency, to keep the text readable of course.

**Comment by dan0xii** [comment]
It looks good with the partially transparent backgrounds and I've added the grey stars. See #471.
